src/serac/numerics/equation_solver.cpp

```diff
@@ -198,11 +204,13 @@ class NewtonSolver : public mfem::NewtonSolver {
 /// Internal structure for storing trust region settings
 struct TrustRegionSettings {
   /// cg tol
-  double cgTol = 1e-8;
+  double cg_tol = 1e-8;
+  /// min cg iters
+  size_t min_cg_iterations = 0;  //
   /// max cg iters should be around # of system dofs
-  size_t maxCgIterations = 10000;  //
+  size_t max_cg_iterations = 10000;  //
   /// max cumulative iterations
-  size_t maxCumulativeIteration = 1;
+  size_t max_cumulative_iteration = 1;
```

Comment:
Is this setting the number of cumulative CG iterations before the precondtioner is updated? If so, I suggest the name be more descriptive, even if it ends up long.